data/server.js

```diff
@@ -0,0 +1,74 @@
+
+
+const express = require('express'); 
+const router = express.Router(); 
+const server = express(); 
+
+const actionDB = require('./helpers/actionModel')
+const projectsDB = require('./helpers/projectModel'); 
+
+
+
+// ACTION req handlers
+
+
+router.get('/', (req, res) => {
+    res.status(201).json({ message: 'hello' }); 
+    actionDB.get()
+        .then(res => res.status(201).json({ api: 'up' }))
+        .catch(err => res.status(500).json({ api: 'down' })); 
+})
+
+router.post('/', (req, res) => { 
+    actionDB.insert()
+        .then(res => res.status(201).json({ api: 'up' }))
+        .catch(err => res.status(500).json({ api: 'down' })); 
+})
+
+router.put('/', (req, res) => {
+    actionDB.update()
+        .then(res => res.status(201).json({ api: 'up' }))
+        .catch(err => res.status(500).json({ api: 'down' })); 
+})
+
+router.delete('/', (req, res) => {
+    actionDB.remove()
+        .then(res => res.status(201).json({ api: 'up' }))
+        .catch(err => res.status(500).json({ api: 'down' })); 
+
+})
+
+
+
+
+
+
+// Projects req handlers
+
+
+router.get('/', (req, res) => {
```

Comment:
same with these thee should be something in the url since you aren't routing from a different file but running the calls/endpoints inside of the server.js